spk/lua/PLIST

```diff
@@ -0,0 +1,7 @@
+lib:lib/libncurses.so.6.*
```

Comment:
Useless file if libraries are declared as `DEPENDS` instead of `BUILD_DEPENDS`

Reply:
I'll declare the libraries as `DEPENDS` instead of `BUILD_DEPENDS` as you requested, but please consider the following:

If I'd move the ncurses dependency from `BUILD_DEPENDS` to `DEPENDS`, all the files from the ncurses library will be incorporated into the package, not just the library file. This will add all sorts of unnecessary files to the package, while only the `libncurses.so` file is needed. IMO this is a bad idea, not only for security reasons, but it also makes the package unnecessary large.

The real problem is though that the latest version of DSM (6.1.5 update 1) is shipped with older versions of libncurses (v5.9) and libreadline (v6.3). spksrc is already updated to v6.0 for libncurses and v7.0 for libreadline. Because of this, any package compiled with spksrc cannot use the default libraries shipped with DSM. Using the default library versions shipped with DSM is the most preferred solution IMO.

As soon as DSM updates libncurses and libreadline to the latest version, packages can use the default libraries. Until then, packages compiled with spksrc have to be shipped with custom libraries.